app/grandchallenge/algorithms/migrations/0043_algorithm_optional_hanging_protocols.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.1.10 on 2023-08-16 13:02
```

Comment:
It is best to delete the new migrations that you're adding and then regenerating them when rebasing on main.

Same comment for the other 3 apps.